index.html

```diff
@@ -2540,18 +2605,438 @@ <h1>
+<code>did:example:1234;service=agent</code>
+              </td>
+              <td>
+A service endpoint URL selected from a DID document.
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+<code>did:example:1234/path</code><br>
+<code>did:example:1234?query=value</code>
+<code>did:example:1234/path?query=value</code><br>
+              </td>
+              <td>
+Another resource as determined by the input DID method and/or DID controller.
+              </td>
+            </tr>
+        </table>
+
+        <p>
+Implementation details and a concrete algorithm for this function can be found
+in [[?DID-RESOLUTION]].
+        </p>
+
+      </section>
+
+    </section>
+
+    <section>
+      <h2>
+Resolver Options
+      </h2>
+
+      <p>
+The <a>DID resolution</a> and <a>DID URL dereferencing</a> functions can
+OPTIONALLY support <dfn>resolver options</dfn> that control how the functions are executed.
+Resolver options are different from DID parameters (see Section
+<a href="#generic-did-url-parameters"></a>) insofar as they are not part of
+the <var>input <a>DID</a></var> or <var>input <a>DID URL</a></var> syntax.
+Resolver options can for example be used to control caching or content negotiation.
+      </p>
+
+      <p>
+<a>Resolver options</a> are expressed as a map of name-value string pairs.
+      </p>
+      <p>
+Common fields are included here. Additional <a>resolver options</a> MAY be method-independent or method-specific.
+      </p>
+      <p>
+The concrete representation of <a>resolver options</a> MUST be defined by the applicable <a>binding</a> of
+the <a>DID resolution</a> or <a>DID URL dereferencing</a> function.
+      </p>
+
+      <p class="note">
+For example, a concrete <a>binding</a> of the <a>DID resolution</a> or <a>DID URL
+dereferencing</a> function might define how resolver options are represented in the form
+of headers of a network protocol.
+      </p>
+
+      <p>
+This specification defines a single concrete <a>resolver option</a>:
+      </p>
+      <ul>
+        <li>
+The <code>accept</code> <var>resolver option</var> can be used to indicate the requested
+MIME type of the representation of the resource that is returned by the <a>DID resolution</a>
+or <a>DID URL dereferencing</a> function.
+          <ul>
+            <li>
+This <var>resolver option</var> is OPTIONAL.
+            </li>
+            <li>
+If this <var>resolver option</var> is supplied to the <a>DID resolution</a> function,
+or if it is supplied to the <a>DID URL dereferencing</a> function and the result is expected
+to be a <a>DID document</a>, then the value of this <var>resolver option</var> MUST be a
+MIME type associated with a compliant representation of the abstract data model for
+<a>DID documents</a> (see Section <a href="#representations"></a>).
+            </li>
+          </ul>
+        </li>
+      </ul>
+
+      <p>
+A <a>DID resolver</a> MAY support additional <var>resolver options</var>.
```

Comment:
probably worth mentioning how they would go about this, and if we should use registries.